### ISSUE TITLE
18s data availability? 

### ISSUE THREAD

**Issue by anna-schrecengost** [biocore/emp#115]
I am working on a project that involves surveying public 18S SSU rRNA gene datasets for specific taxonomic groups and would like to incorporate EMP data if i can. However, I understand that release 1 is all 16S data, and when I look in the Qiita portal I only see 4 18S projects listed. Is there another place to access 18S data that has been generated as a part of the EMP?

**Comment by lukenoaa** [comment]
Hi Anna, you are searching the right place. Those might be the only 4 studies with 18S data from the original EMP. There is a new set of studies (informally called EMP500) that has a lot more 18S data. It's in Qiita, but it's not public yet. Let me see where things are on either publishing or making that data public. - Luke

**Comment by antgonza** [comment]
Note that you could also search/use all 18S Qiita samples, if you wanted to collect more.

**Comment by lukenoaa** [comment]
Good point, @antgonza. The data for any of the 18S studies in Qiita should have been sequenced with the same 18S protocol, and the metadata should follow the same MIxS standard, as the EMP studies.

**Comment by anna-schrecengost** [comment]
Thank you Luke, I appreciate it! And thank you @antgonza , that is a good point

**Comment by anna-schrecengost** [comment]
Hi @lukenoaa, do you have any update on this data availability, or when it will become public?

**Comment by lukenoaa** [comment]
Hi @anna-schrecengost, there's been some progress on that project, but I don't know when it will become public. Did searching all 18S studies in Qiita reveal any additional studies?
